pkg/cmd/preview/preview.go

```diff
@@ -286,24 +286,21 @@ func (o *PreviewOptions) Run() error {
 					"Unable to get commits: %s", err.Error())
 			}
 			if pullRequest != nil {
-				prAuthor := pullRequest.Author
-				if prAuthor != nil {
-					author, err := resolver.Resolve(prAuthor)
-					if err != nil {
-						return err
-					}
-					author, err = resolver.UpdateUserFromPRAuthor(author, pullRequest, commits)
-					if err != nil {
-						// This isn't fatal, just nice to have!
-						log.Logger().Warnf("Unable to update user %s from %s because %v", prAuthor.Name, o.PullRequestName, err)
-					}
-					if author != nil {
-						user = &v1.UserSpec{
-							Username: author.Spec.Login,
-							Name:     author.Spec.Name,
-							ImageURL: author.Spec.AvatarURL,
-							LinkURL:  author.Spec.URL,
-						}
+				author, err := resolver.Resolve(pullRequest.Author)
```

Comment:
Maybe the intention was to ignore a non-set author, as opposed to failing the command. 
Was there another error occurring with the preview command when the nil author was nil?

Reply:
I didn't think of such use case. In that case I should revert the preview.go change entirely.